utils/text2token.py

```diff
@@ -87,22 +87,22 @@ def main():
                 else:
                     break
 
-        if len(match_pos) > 0:
-            chars = []
-            i = 0
-            while i < len(a):
-                start_pos, end_pos = exist_or_not(i, match_pos)
-                if start_pos is not None:
-                    chars.append(a[start_pos:end_pos])
-                    i = end_pos
-                else:
-                    chars.append(a[i])
-                    i += 1
-            a = chars
-
         if(args.trans_type == "phn"):
             a = a.split(" ")
         else:
+            if len(match_pos) > 0:
```

Comment:
Can you add a unit test to validate this change in https://github.com/espnet/espnet/blob/master/test_utils/test_text2token_py.bats?

Reply:
Can you tell me how this works? For example this change is to solve incorrect results when the text contains non-linguistic symbols and the trans_type is phn. So should I add such an example text in the unit test?

Reply:
Yes, I'd like to add the case for the phoneme.

Reply:
I tried to follow https://github.com/espnet/espnet/blob/master/CONTRIBUTING.md#unit-testing but I can not figure out the right way to do it... Can you give a little more details on how to do this? Thank you!